context/ChainContextProvider.tsx

```diff
@@ -48,6 +48,9 @@ const ChainContextProvider: FC<any> = ({ children }) => {
 
       setGlobalChainId(+chainId)
 
+      router.query.chain = supportedChainsMap[+chainId].routePrefix
```

Comment:
Hmm, this doesn't actually work aside from redirecting you to the home page on the chain you clicked on.. Am I missing something?